pkg/service/active_directory_membership_service.go

```diff
@@ -110,7 +110,7 @@ func (svc *ActiveDirectoryMembershipService) DeleteActiveDirectoryMembership(id
 // RemoveActiveDirectoryMembership - leaves the Active Directory domain.
 //   Required parameters:
 //       id - ID of the active directory.
-//       user - Name of the Activer Directory user with the privilege to leave the domain.
+//       user - "Name of the Active Directory user with the privilege to leave the domain."
```

Comment:
The quotes are odd.  I guess it is an artifact of having to override the upstream message to correct a typo.  Do the upstream messages ever change?

Reply:
There is a bug in the upstream . I will fix the quote in next PR. thanks